forge/ee/routes/billing/index.js

```diff
@@ -172,21 +172,14 @@ module.exports = async function (app) {
         const team = request.team
         const sub = await app.db.models.Subscription.byTeam(team.id)
         if (!sub) {
-            if (!request.session.User.admin) {
```

Comment:
This check allowed an Admin user to view a team even if billing wasn't setup. But the UX around this was not very good. Far too often it would lead to confusion as to why a team appeared okay - only to realised you were logged in as admin. The real risk was you would go on to create a project which gets in a bad state because the billing isn't setup properly.